src/api/Volkszaehler.cpp

```diff
@@ -213,7 +213,7 @@ json_object *vz::api::Volkszaehler::api_json_tuples(Buffer::Ptr buf) {
 					// a) timestamp
 					// b) duplicate value
 					if ((timestamp >= (_last_timestamp + duplicates_ms)) ||
-						(r.value() != _lastReadingSent->value())) {
+						(fabs(r.value() - _lastReadingSent->value()) > 0.000001 )) {
```

Comment:
i think the tolerance for the comparison (0.000...01) needs to be chosen in relation to the input values, for this to work with numbers in any range,
but my short research did not lead to a best-practice algorithm to use.
(probably needs a float number with it's exponent derived from the exponent of the inputs.)

Reply:
https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/